functions.php

```diff
@@ -5,83 +5,60 @@
  * @package Go
  */
 
-/**
- * Theme constants.
- */
-define( 'GO_VERSION', '1.8.1' );
+// Load deactivation modal.
+define( 'GO_VERSION', '2.0.0' );
 define( 'GO_PLUGIN_DIR', get_template_directory( __FILE__ ) );
 define( 'GO_PLUGIN_URL', get_template_directory_uri( __FILE__ ) );
+require_once get_parent_theme_file_path( 'includes/class-go-theme-deactivation.php' );
```

Comment:
Minor fix but we should move the comment to be above the require_once line.

Reply:
I put the comment there because that block is only meant for the deactivation modal. I will be removing those defines in a future (and minor) refactor of that class. I wanted to make sure things were working as is for the pipeline before doing that.